### PR TITLE
(API) Oracle Eloqua destination

### PR DESCRIPTION
**Pull request by mohan-h**
Initial commit

### CODE REVIEW

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
+---
+keywords: crm;CRM;crm destinations;api oracle eloqua;api oracle eloqua crm destination
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: As per the documentation self-service template, we don't need to add keywords to the metadata.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+title: (API) Oracle Eloqua connection
+description: The (API) Oracle Eloqua destination allows you to export your account data and activate it within Oracle Eloqua for your business needs.
+exl-id:
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: As per the documentation self-service template, please remove the `exl-id` line. The `exl-id` will be automatically generated after we publish the page. Including this line can actually adversely affect the search experience on the site.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+
+As a marketer, you can deliver personalized experiences to your users, based on attributes from their Adobe Experience Platform profiles. You can build segments from your offline data and send these segments to [!DNL Oracle Eloqua], to display in the users' feeds as soon as segments and profiles are updated in Adobe Experience Platform.
+
+## Prerequisites {#prerequisites}
+
+### Experience Platform prerequisites {#prerequisites-in-experience-platform}
+
+Before activating data to the [!DNL Oracle Eloqua] destination, you must have a [schema](/help/xdm/schema/composition.md), a [dataset](https://experienceleague.adobe.com/docs/platform-learn/tutorials/data-ingestion/create-datasets-and-ingest-data.html?lang=en), and [segments](https://experienceleague.adobe.com/docs/platform-learn/tutorials/segments/create-segments.html?lang=en) created in [!DNL Experience Platform].
+
+Refer to Adobe's documentation for [Segment Membership Details schema field group](/help/xdm/field-groups/profile/segmentation.md) if you need guidance on segment statuses.
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: One comments here: 

Let's please not use Adobe's documentation. Let's use Experience Platform instead, as in: "Refer to the Experience Platform documentation"

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+### [!DNL Oracle Eloqua] prerequisites {#prerequisites-destination}
+
+In order to export data from Platform to your [!DNL Oracle Eloqua] account you need to have an [!DNL Oracle Eloqua] account.
+
+#### Gather [!DNL Oracle Eloqua] credentials {#gather-credentials}
+
+Note down the items below before you authenticate to the [!DNL Oracle Eloqua] destination:
+
+| Credential | Description |
+| --- | --- | --- |
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Please revisit this table, as it renders as broken. The issue is most likely on line 38. 
Please also use a markdown previewer when you author the documentation, to catch these issues before submitting for review. Thanks!

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+| `Username` | The Username of your [!DNL Oracle Eloqua] account. |
+| `Password` | The Password of your [!DNL Oracle Eloqua] account. |
+
+## Guardrails {#guardrails}
+
+>[!NOTE]
+>
+>* When creating contact fields used to save segment mappings, ensure the count falls within the maximum limit of 250 custom contact fields.
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: When creating contact fields -> is this in Eloqua? please call this out clearly. 

Also, we are introducing this concept directly as a limitation -- what are contact fields and why are these needed to save segment mappings?

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+| --- | --- | --- |
+| `Username` | The Username of your [!DNL Oracle Eloqua] account. |
+| `Password` | The Password of your [!DNL Oracle Eloqua] account. |
+
+## Guardrails {#guardrails}
+
+>[!NOTE]
+>
+>* When creating contact fields used to save segment mappings, ensure the count falls within the maximum limit of 250 custom contact fields. 
+>* If the limit is exceeded you will encounter a `400: "There was a validation error"` error.
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Do you encounter this error in Experience Platform or in Eloqua?

Also, a comma is needed before "you".

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+>* To be able to update new segments remove some existing mappings from your datasets and delete the corresponding custom contact fields in your [!DNL Oracle Eloqua] account.
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Please simplify this sentence, something like:

If you have reached the limit specified above, you need to remove existing mappings from your datasets (by the way, why are we talking about datasets??) and delete the corresponding custom contact fields in your [!DNL Oracle Eloqua] account before you can export more segments.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 8]
```diff
@@ -0,0 +1,165 @@
...
+>* Refer to the [Oracle Eloqua Creating Contact Fields](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-user/Help/ContactFields/Tasks/CreatingContactFields.htm) page for information about additional limits.
+
+## Supported identities {#supported-identities}
+
+[!DNL Oracle Eloqua] supports update of identities described in the table below. Learn more about [identities](/help/identity-service/namespaces.md).
+
+|Target Identity|Example|Description|Considerations|
+|---|---|---|---|
+| `EloquaId` | `111111` | Unique identifier of the contact.| **Mandatory**. |
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: No period needed after Mandatory

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 9]
```diff
@@ -0,0 +1,165 @@
...
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Oracle Eloqua] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL Oracle Eloqua] destination fields, follow these steps:
+
+1. In the **[!UICONTROL Mapping]** step, select **[!UICONTROL Add new mapping]**. You will see a new mapping row on the screen.
+1. In the **[!UICONTROL Select source field]** window, choose the **[!UICONTROL Select attributes]** category and select the XDM attribute or choose the **[!UICONTROL Select identity namespace]** and select an identity.
+1. In the **[!UICONTROL Select target field]** window, choose the **[!UICONTROL Select identity namespace]** and select an identity or choose **[!UICONTROL Select custom attributes]** category and select an attribute as needed.
+    * Repeat these steps to add the following mappings between your XDM profile schema and your [!DNL Oracle Eloqua] instance:
+        |Source Field|Target Field| Mandatory|
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md:133]: Some clarification questions: 

1. Are we absolutely sure that **both** these fields (`Attribute: emailAddress` and `Identity: EloquaId`) are absolutely necessary? What happens if a user doesn't select one or both of them?
2. Do the mappings need to be exactly as described in the table? Namely, on the Source Field side, can users select other attributes/identities and map them to the Target fields?

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 10]
```diff
@@ -0,0 +1,165 @@
...
+        |`IdentityMap: Eid`|`Identity: EloquaId`| Yes |
+
+    * An example using these mappings is shown below:
+    ![Platform UI screenshot example with attribute mappings.](../../assets/catalog/email-marketing/oracle-eloqua-api/mappings.png)
+
+When you are finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+>[!NOTE]
+>
+>The destination automatically suffixes a unique identifier to the selected segment names on each execution when sending the contact field information to [!DNL Oracle Eloqua]. This ensures the contact field names corresponding to your segment names do not overlap.
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Can we please provide an example here of what the exported segment with a suffix looks like? or link to somewhere else in the doc where we show an example.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 11]
```diff
@@ -0,0 +1,165 @@
...
+To validate that you have correctly set up the destination, follow the steps below:
+
+1. Select **[!UICONTROL Destinations]** > **[!UICONTROL Browse]** and navigate to the list of destinations.
+1. Next, select the destination and switch to the **[!DNL Activation data]** tab, then select a segment name.
+![Platform UI screenshot example showing Destinations Activation Data.](../../assets/catalog/email-marketing/oracle-eloqua-api/destinations-activation-data.png)
+
+1. Monitor the segment summary and ensure that the count of profiles corresponds to the count within the segment.
+![Platform UI screenshot example showing Segment.](../../assets/catalog/email-marketing/oracle-eloqua-api/segment.png)
+
+1. Log in to the [!DNL Oracle Eloqua] website, then navigate to the [!DNL Contacts Overview] page to check if the profiles from the segment have been added. To see the segment status drill down into a [!DNL Contact Detail] page and check if the contact field having the selected segment name as its prefix has been created.
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: UI elements should be in bold, so that they stand out in the text. this includes UI elements in Eloqua, like [!DNL Contacts Overview] and [!DNL Contact Detail].

minor grammar suggestions: 

To see the segment status, drill down into a [!DNL Contact Detail] page and check if the contact field with the selected segment name as its prefix has been created.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 12]
```diff
@@ -0,0 +1,165 @@
...
+![Oracle Eloqua UI screenshot showing the Contact Details page with contact field created with the segment name.](../../assets/catalog/email-marketing/oracle-eloqua-api/contact.png)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
+
+If the [!DNL Oracle Eloqua] API fails to validate a request, it will respond with a `400: "There was a validation error"` error message describing the issue.
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Minor, but we try to use present tense over future tense. for example: 

If the [!DNL Oracle Eloqua] API fails to validate a request, it responds with a `400: "There was a validation error"` error message describing the issue.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 13]
```diff
@@ -0,0 +1,165 @@
...
+
+First check by ensuring you have met the conditions defined in the (guardrails)[#guardrails] section.
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: As a first step, make sure you have met the conditions outlined in the [guardrails](#guardrails) section.

**Review by vgiurgiu — COMMENTED** [review]
Good work! I left some comments, please let us know if you have any questions. Thanks!

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 2 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Removed however please check the [template](https://github.com/AdobeDocs/experience-platform.en/blob/main/help/destinations/destination-sdk/docs-framework/self-service-template.md) still has the exl-id.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 5 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Have reworded.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 6 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Eloqua, have reworded. 
Thanks for the comma catch.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 7 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Reworded.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 10 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Reworded and added reference to the validation section.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 11 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Thanks

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 13 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Updated.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 9 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md:133]: _Are we absolutely sure that both these fields (Attribute: emailAddress and Identity: EloquaId) are absolutely necessary? What happens if a user doesn't select one or both of them?_

Dev team : Yes ,EloquaID is needed in the http call as corresponding to that all other attributes are updated, Email Id is also necessary as without it the API throws error.

```
    {
        "type": "ObjectValidationError",
        "container": {
            "type": "ObjectKey",
            "objectType": "Contact"
        },
        "property": "emailAddress",
        "requirement": {
            "type": "EmailAddressRequirement"
        },
        "value": "<null>"
    }
]  
```

_Do the mappings need to be exactly as described in the table? Namely, on the Source Field side, can users select other attributes/identities and map them to the Target fields?_

No, All we need to consider is the target mapping. The attributes need to be named as described in table because these attributes will form request body.
From source side we don't have any restriction to map fields ,you can map from any where you want if the data is appropriate it will be pushed to destination else will throw an error.
(Example - you can map EloquaID to any identity namespace like contact key,outreach ID etc. but if that ID is not present in Eloqua DB it will throw an error.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 2 above)
Comment (vgiurgiu) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Ah, I see what happened here. The link you provide above is the Got location for the published example page on Experience League. Since this is a live page, it will have an `exl-id`. 

For the latest template, please always go to this link (https://experienceleague.adobe.com/docs/experience-platform/destinations/destination-sdk/document-destination/documentation-instructions.html?lang=en#documentation-template) and select **Download the template here** to get the latest version of the template, which does not have an `exl-id`.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 14]
```diff
@@ -0,0 +1,154 @@
...
+| --- | --- |
+| `Username` | The Username of your [!DNL Oracle Eloqua] account. |
+| `Password` | The Password of your [!DNL Oracle Eloqua] account. |
+
+## Guardrails {#guardrails}
+
+>[!NOTE]
+>
+>* [!DNL Oracle Eloqua] has a maximum limit of 250 custom contact fields.
+>* [!DNL Oracle Eloqua] custom contact fields are created using the names of the segments selected during the **[!UICONTROL Destinations]** > **[!UICONTROL Activate Destination]** > **[!UICONTROL Select segments]** step.
```
Comment (vgiurgiu) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Thanks for updating this part. I just have a follow-up question. Are [!DNL Oracle Eloqua] custom contact fields created automatically in Eloqua or does the user need to create these? 

If these are created automatically, my suggestion is to add this callout to make things as clear as possible.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 9 above)
Comment (vgiurgiu) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md:133]: These are great clarifications, thanks!

Could we transform these clarifications into material that we could add to this documentation page?

**Review by vgiurgiu — COMMENTED** [review]
Thanks for the updates, I left two more comments, looking for some minor clarifications.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 14 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: They are created automatically, have reworded.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 9 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md:133]: Sure, incorporated.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 15]
```diff
@@ -0,0 +1,183 @@
+---
+title: (API) Oracle Eloqua connection
+description: The (API) Oracle Eloqua destination allows you to export your account data and activate it within Oracle Eloqua for your business needs.
+---
+# [!DNL (API) Oracle Eloqua] connection
+
+[[!DNL Oracle Eloqua]](https://www.oracle.com/cx/marketing/automation/) enables marketers to plan and execute campaigns while delivering a personalized customer experience for their prospects. With integrated lead management and easy campaign creation, it helps marketers engage the right audience at the right time in their buyer's journey and elegantly scales to reach audiences across channels including email, display search, video, and mobile. Sales teams can close more deals at a faster rate, increasing marketing ROI through real-time insight.
+
+This [!DNL Adobe Experience Platform] [destination](/help/destinations/home.md) leverages the [[!DNL Oracle Eloqua] Update a Contact REST API](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/op-api-rest-1.0-data-contact-id-put.html), which allows you to update identities within a segment into [!DNL Oracle Eloqua].
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: "Update a contact" is just an operation supported by their REST API, it's not an API itself.
 Let's rephrase that to "[...] leverages the [Update a contact](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/op-api-rest-1.0-data-contact-id-put.html) operation from the Oracle Eloqua REST API [...]"

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 16]
```diff
@@ -0,0 +1,183 @@
...
+
+[!DNL Oracle Eloqua] uses [Basic Authentication](https://docs.oracle.com/en/cloud/saas/marketing/eloqua-rest-api/Authentication_Basic.html) to communicate with the [!DNL Oracle Eloqua Update a Contact REST API]. Instructions to authenticate to your [!DNL Oracle Eloqua] instance are further below, in the [Authenticate to destination](#authenticate) section.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: "to communicate with the [!DNL Oracle Eloqua REST API]"

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 17]
```diff
@@ -0,0 +1,183 @@
...
+
+In order to export data from Platform to your [!DNL Oracle Eloqua] account you need to have an [!DNL Oracle Eloqua] account.
+
+#### Gather [!DNL Oracle Eloqua] credentials {#gather-credentials}
+
+Note down the items below before you authenticate to the [!DNL Oracle Eloqua] destination:
+
+| Credential | Description |
+| --- | --- |
+| `Username` | The Username of your [!DNL Oracle Eloqua] account. |
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Username and password in the Description column should be lowercase.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 18]
```diff
@@ -0,0 +1,183 @@
...
+| `Password` | The Password of your [!DNL Oracle Eloqua] account. |
+
+## Guardrails {#guardrails}
+
+>[!NOTE]
+>
+>* [!DNL Oracle Eloqua] custom contact fields are automatically created using the names of the segments selected during the **[!UICONTROL Destinations]** > **[!UICONTROL Activate Destination]** > **[!UICONTROL Select segments]** step.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: No need to specify the entire path to a step when referring to said step. Please rephrase as "[!DNL Oracle Eloqua] custom contact fields are automatically created using the names of the segments selected during the  **[!UICONTROL Select segments]** step."

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 19]
```diff
@@ -0,0 +1,183 @@
...
+
+* [!DNL Oracle Eloqua] has a maximum limit of 250 custom contact fields.
+* Before exporting new segments ensure you are within this limit by logging into your [!DNL Oracle Eloqua] account.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: This part is confusing. You're saying that Eloqua has a maximum limit of 250 custom contact fields. Then you ask readers to make sure they are within the limit, by checking their Eloqua account. I don't see how this makes sense. If the limit is on Eloqua's side, you will always be within that limit. That's what a limit does, right?

I believe what you meant here is to ask readers to check their Platform account and make sure that the number of segments they are exporting + the number of existing segments in Eloqua do not exceed the limit. We should be very clear on this behavior.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 20]
```diff
@@ -0,0 +1,183 @@
...
+>To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+Within **[!UICONTROL Destinations]** > **[!UICONTROL Catalog]** search for [!DNL (API) Oracle Eloqua]. Alternatively you can locate it under the **[!UICONTROL Email Marketing]** category.
+
+### Authenticate to destination {#authenticate}
+
+Fill in the required fields below. Refer to the [Gather [!DNL Oracle Eloqua] credentials](#gather-credentials) section for any guidance.
+* **[!UICONTROL Password]**: The Password of your [!DNL Oracle Eloqua] account.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Password and username in lowercase in the description.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 21]
```diff
@@ -0,0 +1,183 @@
...
+
+Read [Activate profiles and segments to streaming segment export destinations](/help/destinations/ui/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Oracle Eloqua] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. 
+
+>[!IMPORTANT]
+>
+>Both the attribute mappings listed in the table which follows are mandatory.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Which table? There is no table here.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 22]
```diff
@@ -0,0 +1,183 @@
...
+        |`IdentityMap: Eid`|`Identity: EloquaId`| Yes |
+
+    * An example using these mappings is shown below:
+    ![Platform UI screenshot example with attribute mappings.](../../assets/catalog/email-marketing/oracle-eloqua-api/mappings.png)
+
+When you are finished providing the mappings for your destination connection, select **[!UICONTROL Next]**.
+
+>[!NOTE]
+>
+>The destination automatically suffixes a unique identifier to the selected segment names on each execution when sending the contact field information to [!DNL Oracle Eloqua]. This ensures the contact field names corresponding to your segment names do not overlap. Refer to the [Validate data export](#exported-data) section screenshot example of a [!DNL Oracle Eloqua] Contact Details page with custom contact field created using the segment names.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: "**an** [!DNL Oracle Eloqua] Contact Details page"

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md [hunk 23]
```diff
@@ -0,0 +1,183 @@
...
+
+![Oracle Eloqua UI screenshot showing the Contact Details page with custom contact field created with the segment name.](../../assets/catalog/email-marketing/oracle-eloqua-api/contact.png)
+
+## Data usage and governance {#data-usage-governance}
+
+All [!DNL Adobe Experience Platform] destinations are compliant with data usage policies when handling your data. For detailed information on how [!DNL Adobe Experience Platform] enforces data governance, see the [Data Governance overview](/help/data-governance/home.md).
+
+## Errors and troubleshooting {#errors-and-troubleshooting}
+
+When creating the destination, if you obtain an error message: `400: There was a validation error` or `400 BAD_REQUEST` as the [!DNL Oracle Eloqua] API response.
```
Comment (vfalon) [inline help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: The phrasing here sounds incomplete/incorrect. Let's rephrase this.

"When creating the destination, you might receive one of the following error messages: `400: There was a validation error` or `400 BAD_REQUEST`. This happens when you exceed the 250 custom contact fields limit, as described in the [guardrails](#guardrails) section. To fix this error, make sure you are not exceeding the limit.

**Review by vfalon — CHANGES_REQUESTED** [review]
I've left a few comments to address. Let me know if you have any questions.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 21 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Changed the position to just below the table.

File: help/destinations/catalog/email-marketing/oracle-eloqua-api.md (hunk 19 above)
Comment (mohan-h) [reply help/destinations/catalog/email-marketing/oracle-eloqua-api.md]: Thanks, have rephrased
